sql_generators/mobile_kpi_support_metrics/__init__.py

```diff
@@ -19,13 +19,15 @@
 HEADER = f"-- Query generated via `{GENERATOR_ROOT.name}` SQL generator."
 VERSION = "v1"
 TEMPLATES = (
-    "active_users.view.sql",
-    "retention_clients.view.sql",
-    "retention.query.sql",
-    "retention.view.sql",
-    "engagement_clients.view.sql",
-    "engagement.query.sql",
-    "engagement.view.sql",
+    # "active_users.view.sql",
+    # "retention_clients.view.sql",
+    # "retention.query.sql",
+    # "retention.view.sql",
+    # "engagement_clients.view.sql",
+    # "engagement.query.sql",
+    # "engagement.view.sql",
```

Comment:
are you going to uncomment before merging?

Reply:
yes, that's my bad. Thanks